web/wp-content/themes/dctx2/.eslintrc.js

```diff
@@ -0,0 +1,535 @@
+		 *
+		 * @author Aubrey Portwood
+		 * @since 1.0
+		 */
+		'space-before-blocks': [ 'error', 'always' ],
+
+		/**
+		 * Require or disallow space before function opening parenthesis.
+		 *
+		 * @standard WP
+		 * @see      https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing
+		 *
+		 * @author Aubrey Portwood
+		 * @since 1.0
+		 */
+		'space-before-function-paren': [ 'error', 'never' ],
+
+		/**
+		 * Require or disallow space before blocks.
+		 *
+		 * @standard WP
+		 * @see      https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing
+		 *
+		 * @author Aubrey Portwood
+		 * @since  1.1
+		 */
+		'space-in-parens': [ 'error', 'always' ],
+
+		/**
+		 * Require spaces around operators.
+		 *
+		 * @standard WP
+		 * @see      https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing
+		 *
+		 * @author Aubrey Portwood
+		 * @since 1.1
+		 */
+		'space-infix-ops': 'error',
+
+		/**
+		 * Require or disallow spaces before/after unary operators (words on by default, nonwords),
+		 *
+		 * @standard WP
+		 *
+		 * @author Aubrey Portwood
+		 * @since  1.1
+		 */
+		'space-unary-ops': [ 'error', {
+			overrides: { '!': true }
+		} ],
+
+		/**
+		 * Don't force vars to be on top.
+		 *
+		 * In contradiction to WP Coding Standards,
+		 * we do not require this.
+		 *
+		 * @standard WDS
+		 * @see      https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#declaring-variables-with-var
+		 *
+		 * @since  1.1
+		 * @author Aubrey Portwood
+		 */
+		'vars-on-top': 'off',
+
+		/**
+		 * Require or disallow Yoda conditions.
+		 *
+		 * @standard WP
+		 * @see      https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#yoda-conditions
+		 *
+		 * @since  1.0
+		 * @author Aubrey Portwood
+		 */
+		'yoda': [ 'error', 'always' ],
+
+		/**
+		 * Always show an error when a variable is created that is never used.
+		 *
+		 * @standard WP
+		 *
+		 * @author Aubrey Portwood
+		 * @since  1.1
+		 */
+		'no-unused-vars': 'error',
+
+		/**
+		 * No use of console.
+		 *
+		 * Use of console can be done safely with checking
+		 * window.console and running it from there.
+		 *
+		 * @standard WDS
+		 *
+		 * @author Aubrey Portwood
+		 * @since  1.1
+		 */
+		'no-console': 'error',
+
+		/**
+		 * No use of debugger.
+		 *
+		 * This is because we often can leave it in the code,
+		 * this draws a nice red line around it.
+		 *
+		 * @standard WDS
+		 *
+		 * @since  1.1
+		 * @author Aubrey Portwood
+		 */
+		'no-debugger': 'error',
+
+		/**
+		 * Require valid jsdoc blocks.
+		 *
+		 * @standard WDS
+		 *
+		 * @since  1.1
+		 * @author Aubrey Portwood
+		 */
+		'valid-jsdoc': [ 'error', {
+
+			// If and only if the function or method has a return statement (this option value does apply to constructors)
```

Comment:
:no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).